nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -363,6 +363,8 @@ In addition to numerous new and updated packages, this release has the following
 
 - The `nerdfonts` package has been updated to major version 3, which includes potential [breaking changes](https://github.com/ryanoasis/nerd-fonts/releases/tag/v3.0.0).
 
+- The `html-proofer` package has been updated from major version 3 to major version 5, which includes [breaking changes](https://github.com/gjtorikian/html-proofer/blob/v5.0.8/UPGRADING.md).
```

Comment:
1. write this in rl-2311
2. Split in a separate commit, with title  `rl-2311: mention upgrade of html-proofer to major version 5`

Reply:
Ah I'm making a bunch of noob mistake. I appreciate the quick review though :+1: 

lmk if anything else should be changed